cmd/kubeadm/app/cmd/reset.go

```diff
@@ -97,7 +97,7 @@ func (r *Reset) Run(out io.Writer) error {
 
 	// Try to unmount mounted directories under /var/lib/kubelet in order to be able to remove the /var/lib/kubelet directory later
 	fmt.Printf("[reset] Unmounting mounted directories in %q\n", "/var/lib/kubelet")
-	umountDirsCmd := "cat /proc/mounts | awk '{print $2}' | grep '/var/lib/kubelet' | xargs -r umount"
+	umountDirsCmd := `awk '$2 ~ /\/var\/lib\/kubelet/ {print $2}' /proc/mounts | xargs -r umount`
```

Comment:
You can avoid the escaping by using a dynamic regex variable. e.g. `awk '$2~path {print $2}' path="/var/lib/kubelet" /proc/mounts`.